src/reporter.ts

```diff
@@ -125,10 +135,24 @@ class RPReporter implements Reporter {
     }
   }
 
+  setStatus(status: string, test: TestCase, suite: string): void {
```

Comment:
Method `setStatus` should accept only statuses from  type RP_STATUSES, not custom or just string